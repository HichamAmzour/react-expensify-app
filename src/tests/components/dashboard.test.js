import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashBoard from '../../Components/ExpenseDashboard';

test("should render expense Dashboard correctly !", ()=>{
    const wrapper= shallow(<ExpenseDashBoard />)
    expect(wrapper).toMatchSnapshot()
})