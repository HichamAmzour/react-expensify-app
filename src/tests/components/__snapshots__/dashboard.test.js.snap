// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense Dashboard correctly ! 1`] = `
<div>
  <p>
    this is Expense Dashboard component
  </p>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
